config/sidekiq.yml

```diff
@@ -22,8 +22,8 @@
     # Online converter: https://crontab.guru/#0_22_*_*_*
     #
     RefreshAppendix5aGuidanceWorker:
-      cron: "00 22 * * *"
-      description: "Hot refreshes Chief CDS Guidance"
+      cron: "00 22 * * 1,2,3,4,5"
```

Comment:
Some of the other blocks have a comment with what the timing is. I know I struggle to interpret cron timings so it could be helpful.